backend/tracim_backend/templates/mail/upload_permission_to_emitter_body_html.mak

```diff
@@ -14,14 +14,13 @@
 %endif
 
 <p>${_("Enjoy your day :)")}</p>
-<p>${_("Suricat', your digital assistant")}</p>
 
-<pre>
---
-${_("Suricat', the bot")}
-${config.WEBSITE__TITLE}
-<a href="${config.WEBSITE__BASE_URL}">${config.WEBSITE__BASE_URL}</a>
-${_("powered by tracim software")} - <a href="https://www.tracim.fr">https://www.tracim.fr</a>
-</pre>
+<p style="font-size: 0.8em; color: #5e5c64">
+  --<br/>
+  ${_("You are receiving this email because you have an account on {website_title}.").format(website_title=config.WEBSITE__TITLE)}<br />
+  ${_('You can change this setting by going to the "My Spaces" menu through this link:')}
+  <a href="${config.WEBSITE__BASE_URL}/ui/account">${config.WEBSITE__BASE_URL}/ui/account</a>.<br/>
+  ${_("This e-mail was automatically generated by the")} <a href="https://www.tracim.fr">Tracim software</a> ${_("that powers {website_title}, please do not reply.").format(website_title=config.WEBSITE__TITLE)}
+</p>
```

Comment:
There is a lot of repetition, not very familiar with Mako (if that is what is used) but couldn't this be in a separated file so that it can be imported and reused instead of declaring it every time ?

Reply:
This is an improvement, I suggest to create an issue in order to implement it later. (During the summer, I think)